addons/cookoff/XEH_postInit.sqf

```diff
@@ -11,6 +11,16 @@
 [QGVAR(smoke), FUNC(smoke)] call CBA_fnc_addEventHandler;
 [QGVAR(cookOffBox), FUNC(cookOffBox)] call CBA_fnc_addEventHandler;
 
+// handle cleaning up effects when vehicle is deleted mid-cookoff 
+[QGVAR(addCleanupHandlers), {
+    params ["_vehicle"];
```

Comment:
There should be a guard to prevent the EH from being added multiple times, or the EH should be removed after cook off is done. Vehicle can cook off again if it's reparied/rearmed.